keyboards/converter/ibm_terminal/keymaps/default/rules.mk

```diff
@@ -1,20 +1,21 @@
 # Build Options
-#   change to "no" to disable the options, or define them in the Makefile in 
+#   change to "no" to disable the options, or define them in the Makefile in
 #   the appropriate keymap folder that will get included automatically
 #
 BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
-MOUSEKEY_ENABLE = yes        # Mouse keys
+MOUSEKEY_ENABLE = no        # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no         # Commands for debug and configuration
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
```

Comment:
Honestly, I'm not a fan of these changes. 

Mousekeys may be more useful to people.  And console and command are really mostly only useful for debugging.  

This change (and the matching one to the keyboard level rules.mk) should be reverted.

Reply:
These got in by accident, I wasn't meant to change this